specs/commandline/verify.md

```diff
@@ -197,3 +197,39 @@ export NOTATION_EXPERIMENTAL=1
 # The value of --scope should be set base on the trust policy configuration
 notation verify --oci-layout --scope "local/hello-world" hello-world:v1
 ```
+
+### Verify signatures on an OCI artifact with json output
+
+Use the `--output` flag to format verification output in json.
```

Comment:
Can you also add this new flag along with its description into the `Outline` session of this spec? (https://github.com/notaryproject/notation/blob/main/specs/commandline/verify.md#outline)